app/HutchManager/Controllers/AccountController.cs

```diff
@@ -109,6 +114,12 @@ public async Task<IActionResult> Login(LoginModel model)
           throw new InvalidOperationException(
             $"Successfully signed in user could not be retrieved! Username: {model.Username}");
 
+        if (_loginOptions.RequireConfirmedAccount)
+        {
+          if (!user.AccountConfirmed) // check if AccountConfirmed is false
+            return BadRequest(new LoginResult{IsUnconfirmedAccount = true});
+        }
+        
```

Comment:
```suggestion
        if (_loginOptions.RequireConfirmedAccount && !user.AccountConfirmed)
        {
          // check if AccountConfirmed is false
          return BadRequest(new LoginResult{IsUnconfirmedAccount = true});
        }
        
```
These two ifs can be combined like this